### DISCUSSION TITLE
[QUESTION] Is there a way to relocate config file in windows not in appdata

### DISCUSSION THREAD

**Discussion by Tartifless** [cxong/cdogs-sdl#871] [Q&A] [answered]
Hello,
Using the portable (.zip) version 2.2.0, i'd like to keep my options.cnf and autosave.json either close to the exe or in a place of my choice.
Is there a way to do this ?

**Comment by cxong** [comment, marked as answer]
You can set the environment variable `CDOGS_CONFIG_DIR`, although it seems this is undocumented and I don't know if it works. Try it out and let us know if it does!

**Comment by Tartifless** [comment]
So, i have set the variable to C:\TEST\C-Dogs.SDL-2.2.0-win32 and it created a new folder and stored config within C:\TEST\C-Dogs SDL

I then tried to add a \ at the end : C:\TEST\C-Dogs.SDL-2.2.0-win32\
It then created the folder C:\TEST\C-Dogs.SDL-2.2.0-win32\C-Dogs SDL

So it works correctly, one just needs to think about the final \ !

**Reply by cxong** [reply]
that is interesting and confusing; created a bug ticket for it thanks for the find https://github.com/cxong/cdogs-sdl/issues/872
